sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -893,6 +893,16 @@ object SQLConf {
       .booleanConf
       .createWithDefault(false)
 
+  val THRIFTSERVER_QUERY_TIMEOUT =
+    buildConf("spark.sql.thriftServer.queryTimeout")
+      .doc("Specifies a global timeout value for Thrift Server. A query will be cancelled " +
+        "automatically after the specified time. If a timeout value is set for each statement " +
+        "(e.g., via `java.sql.Statement.setQueryTimeout`), the value takes precedence. " +
+        "If the value is zero or negative, no timeout happens.")
```

Comment:
Could we add time unit to doc? e.g., 'Timeout in seconds ...'

Reply:
okay.

Reply:
Updated, could you check this again?